package.json

```diff
@@ -18,9 +18,9 @@
     "immutable": "^3.8.2",
     "node-sass": "^4.12.0",
     "prop-types": "^15.6.2",
-    "react": "^16.11.0",
-    "react-codemirror2": "^4.2.1",
-    "react-dom": "^16.11.0",
+    "react": "^16.10.2",
+    "react-codemirror2": "^6.0.0",
+    "react-dom": "^16.10.2",
```

Comment:
These are the wrong react versions - should keep `"react": "^16.11.0"` and `"react-dom": "^16.11.0"`

Reply:
my b missed the 10->11, fixed